via/CMakeLists.txt

```diff
@@ -68,17 +68,14 @@ endif()
 
 find_package(jsoncpp CONFIG)
 
-if (TARGET jsoncpp_static)
-    # RPATH causes issues for the Linux SDK tarball due to absolute paths.
-    # This can be worked around with relative RPATHS but the tarball doesn't ship jsoncpp.
-    # So the easiest solution is to just the static version of jsoncpp.
-    target_link_libraries(vkvia PRIVATE jsoncpp_static)
-    set_target_properties(vkvia PROPERTIES SKIP_BUILD_RPATH TRUE)
-elseif(UNIX)
-    # https://github.com/LunarG/VulkanTools/issues/1908
-    find_package(PkgConfig REQUIRED)
+# https://github.com/LunarG/VulkanTools/issues/1908
+find_package(PkgConfig)
+if (PKG_CONFIG_FOUND)
     pkg_check_modules(jsoncpp REQUIRED IMPORTED_TARGET jsoncpp)
     target_link_libraries(vkvia PRIVATE PkgConfig::jsoncpp)
+else()
+    find_package(jsoncpp CONFIG)
+    target_link_libraries(vkvia PRIVATE jsoncpp_static)
```

Comment:
Note this part is important for internal Windows build that doesn't have pkg-config. We currently don't use pkg-config at all for our Windows builds and adding it is low priority.